### ISSUE TITLE
Crash OSM Live without internet connection

### ISSUE THREAD

**Issue by CJTmmr** [osmandapp/Osmand#2169]
with the latest nightly's :
OsmAnd crashes immediately if you have no internet connection and you choose OSM Live from the drawer

On Samsung Galaxy Camera 2, android 4.3

**Comment by quelbs** [comment]
Same problem for me. Attached my exception log:

```
Version  OsmAnd~ 2.3.0#15318M
18.01.2016 8:29:55
Apk Version : 2.3.0#15318M 231
Exception occured in thread Thread[main,5,main] : 
java.lang.NullPointerException
	at net.osmand.plus.liveupdates.ReportsFragment$2.onResponse(ReportsFragment.java:106)
	at net.osmand.plus.liveupdates.ReportsFragment$2.onResponse(ReportsFragment.java:102)
	at net.osmand.plus.liveupdates.ReportsFragment$GetJsonAsyncTask.onPostExecute(ReportsFragment.java:248)
	at android.os.AsyncTask.finish(AsyncTask.java:631)
	at android.os.AsyncTask.access$600(AsyncTask.java:177)
	at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:644)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:153)
	at android.app.ActivityThread.main(ActivityThread.java:5299)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:511)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
	at dalvik.system.NativeStart.main(Native Method)
Version  OsmAnd~ 2.3.0#15318M
18.01.2016 8:29:55
Apk Version : 2.3.0#15318M 231
Exception occured in thread Thread[main,5,main] : 
java.lang.NullPointerException
	at net.osmand.plus.liveupdates.ReportsFragment$2.onResponse(ReportsFragment.java:106)
	at net.osmand.plus.liveupdates.ReportsFragment$2.onResponse(ReportsFragment.java:102)
	at net.osmand.plus.liveupdates.ReportsFragment$GetJsonAsyncTask.onPostExecute(ReportsFragment.java:248)
	at android.os.AsyncTask.finish(AsyncTask.java:631)
	at android.os.AsyncTask.access$600(AsyncTask.java:177)
	at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:644)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:153)
	at android.app.ActivityThread.main(ActivityThread.java:5299)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:511)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
	at dalvik.system.NativeStart.main(Native Method)
```

**Comment by GaidamakUA** [comment]
Should be fixed with commit e4d3286

**Comment by CJTmmr** [comment]
I can confirm : with latest nightly, no more crashes
Op 18 jan. 2016 15:31 schreef "Yurii Mazurevich" <notifications@github.com>:
